legend-engine-xt-persistence-component/legend-engine-xt-persistence-component-relational-core/pom.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ Copyright 2022 Goldman Sachs
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <groupId>org.finos.legend.engine</groupId>
+        <artifactId>legend-engine-xt-persistence-component</artifactId>
+        <version>3.15.4-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>legend-engine-xt-persistence-component-relational-core</artifactId>
+    <packaging>jar</packaging>
+    <name>Legend Engine - XT - Persistence - Component - Relational Core</name>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.finos.legend.engine</groupId>
+            <artifactId>legend-engine-xt-persistence-component-logical-plan</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.finos.legend.engine</groupId>
+            <artifactId>legend-engine-xt-persistence-component-physical-plan</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+
+        <!-- IMMUTABLES -->
+        <dependency>
+            <groupId>org.immutables</groupId>
+            <artifactId>value</artifactId>
+        </dependency>
+
+        <!-- LOGGING -->
+        <dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>slf4j-api</artifactId>
+        </dependency>
+
+        <!-- TEST -->
+        <dependency>
+            <groupId>org.junit.jupiter</groupId>
+            <artifactId>junit-jupiter-api</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.junit.jupiter</groupId>
+            <artifactId>junit-jupiter-engine</artifactId>
```

Comment:
Need test scope

Reply:
Scope should be inherited from the parent pom dependency management section

Reply:
david@Colossus:~/Code/IDEA/legend-engine-fork/legend-engine-xt-persistence-component/legend-engine-xt-persistence-component-physical-plan$ mvn dependency:tree
[INFO] Scanning for projects...
[INFO] Inspecting build with total of 1 modules...
[INFO] Installing Nexus Staging features:
[INFO]   ... total of 1 executions of maven-deploy-plugin replaced with nexus-staging-maven-plugin
[INFO] 
[INFO] --< org.finos.legend.engine:legend-engine-xt-persistence-component-physical-plan >--
[INFO] Building Legend Engine - XT - Persistence - Component - Physical Plan 3.15.4-SNAPSHOT
[INFO] --------------------------------[ jar ]---------------------------------
Downloading from ossrh: https://s01.oss.sonatype.org/content/repositories/snapshots/org/finos/legend/engine/legend-engine-xt-persistence-component-logical-plan/3.15.4-SNAPSHOT/maven-metadata.xml
Downloading from ossrh-releases: https://s01.oss.sonatype.org/content/repositories/releases/org/finos/legend/engine/legend-engine-xt-persistence-component-logical-plan/3.15.4-SNAPSHOT/maven-metadata.xml
Downloading from ossrh-releases: https://s01.oss.sonatype.org/content/repositories/releases/org/finos/legend/engine/legend-engine-xt-persistence-component/3.15.4-SNAPSHOT/maven-metadata.xml
Downloading from ossrh: https://s01.oss.sonatype.org/content/repositories/snapshots/org/finos/legend/engine/legend-engine-xt-persistence-component/3.15.4-SNAPSHOT/maven-metadata.xml
Downloading from ossrh-releases: https://s01.oss.sonatype.org/content/repositories/releases/org/finos/legend/engine/legend-engine/3.15.4-SNAPSHOT/maven-metadata.xml
Downloading from ossrh: https://s01.oss.sonatype.org/content/repositories/snapshots/org/finos/legend/engine/legend-engine/3.15.4-SNAPSHOT/maven-metadata.xml
Downloaded from ossrh: https://s01.oss.sonatype.org/content/repositories/snapshots/org/finos/legend/engine/legend-engine/3.15.4-SNAPSHOT/maven-metadata.xml (615 B at 8.7 kB/s)
[INFO] 
[INFO] --- maven-dependency-plugin:2.10:tree (default-cli) @ legend-engine-xt-persistence-component-physical-plan ---
[INFO] org.finos.legend.engine:legend-engine-xt-persistence-component-physical-plan:jar:3.15.4-SNAPSHOT
[INFO] +- org.finos.legend.engine:legend-engine-xt-persistence-component-logical-plan:jar:3.15.4-SNAPSHOT:compile
[INFO] +- org.immutables:value:jar:2.8.2:provided
[INFO] +- com.fasterxml.jackson.core:jackson-core:jar:2.10.5:compile
[INFO] +- com.fasterxml.jackson.core:jackson-databind:jar:2.10.5.1:compile
[INFO] |  \- com.fasterxml.jackson.core:jackson-annotations:jar:2.10.5:compile
[INFO] +- commons-codec:commons-codec:jar:1.15:compile
[INFO] +- org.junit.jupiter:junit-jupiter-api:jar:5.4.0:test
[INFO] |  +- org.apiguardian:apiguardian-api:jar:1.0.0:test
[INFO] |  +- org.opentest4j:opentest4j:jar:1.1.1:test
[INFO] |  \- org.junit.platform:junit-platform-commons:jar:1.4.0:test
[INFO] \- org.junit.jupiter:junit-jupiter-engine:jar:5.4.0:test
[INFO]    \- org.junit.platform:junit-platform-engine:jar:1.4.0:test
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  1.469 s
[INFO] Finished at: 2022-09-14T11:15:48-04:00
[INFO] ------------------------------------------------------------------------

Reply:
Verified via CLI just to be sure